source/package.lisp

```diff
@@ -1,20 +1,27 @@
 (defpackage ccldoc
+  (:use ::cl)
```

Comment:
Why two colons in `::cl`?

This may not matter, but ccl's default use clause is (:use :cl :ccl).